test_runner/fixtures/neon_fixtures.py

```diff
@@ -1762,15 +1763,15 @@ def __init__(self, env: NeonEnv, port: PageserverPort, config_override: Optional
             ".*Removing intermediate uninit mark file.*",
         ]
 
-    def start(self, overrides: Tuple[str, ...] = ()) -> "NeonPageserver":
+    def start(self, overrides: Tuple[str, ...] = (), extra_env_vars={}) -> "NeonPageserver":
```

Comment:
it seems that there were updates from @hlinnaka

Reply:
Addressed by my force push https://github.com/neondatabase/neon/pull/2861#issuecomment-1322029016